src/app/auth/authentication.service.ts

```diff
@@ -61,7 +61,9 @@ export class AuthenticationService {
       let refreshInMs = Math.round(refreshInSeconds * .9) * 1000;
       console.log('Refreshing token in: ' + refreshInMs + ' milliseconds.');
       this.refreshInterval = refreshInMs;
-      this.clearTimeoutId = setTimeout(() => this.refreshToken(), refreshInMs);
+      if (process.env.ENV !== 'inmemory') {
```

Comment:
This is being used in planner. It's causing protractor failing to understand the APP load completion, thus functional tests are failing with timeout.